tests/drivers/charger/sbs_charger/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
please add tests into a separate commit to ease review

Reply:
sure, I can split it into two commits:
- Introduces the driver
- Adds tests

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/56666#discussion_r1173067367
I don't want to mess up the existing comments by splitting up the commits. I'll split it up once this comment is resolved.